diagnostic.md

```diff
@@ -150,7 +196,11 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne would display on the right side of the page
+divTwo would display to the left of divOne
+divThe and divFour would be stacked vertically beneath divTwo:
+  divThree would display beneath divTwo
+  divFour would display beneath divThree
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5